### DISCUSSION TITLE
一个关于B站账号的问题

### DISCUSSION THREAD

**Discussion by NightSharp** [openvpi/DiffSinger#46] [General]
请问一下作者大佬
为什么B站的[夜揽青河](https://space.bilibili.com/65682323)为什么会在您B站账号[YQ之神](https://space.bilibili.com/102844209)的黑名单中，印象中之前应该没有过什么交集
@yqzhishen

**Comment by yqzhishen** [comment]
看了一下可能是在2020年或者2021年左右的时期我在某个视频的评论区对观点相左的人进行预防性拉黑的时候加入黑名单的，具体的情况我已经记不清了。
现在已经从黑名单中移除。

**Reply by NightSharp** [reply]
感谢
